torch/distributed/tensor/debug/_visualize_sharding.py

```diff
@@ -79,8 +79,8 @@ def _create_rich_table(
     import rich.style
     import rich.table
 
-    dtensor_height = shape[0] if len(shape) > 0 else 1
-    dtensor_width = shape[1] if len(shape) > 0 else shape[0]
+    dtensor_height = shape[0]
+    dtensor_width = shape[1] if len(shape) == 2 else 1
```

Comment:
Here I am fixing a bug that I created in the previous PR. When the tensor is 1D, consider it a column vector.